system/Session/Handlers/BaseHandler.php

```diff
@@ -81,7 +81,7 @@ abstract class BaseHandler implements SessionHandlerInterface
     /**
      * Current session ID
      *
-     * @var string
+     * @var string|null
```

Comment:
This shuts up phpstan errors.  `$sessionID` CAN be null until we `read()` the session data, so checking it via `isset()` is valid.